fireplace/ai/examples/one_ai_player_demo.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+import asyncio
+import logging
+import threading
+import time
+from fireplace import cards
+from fireplace.game import Game
+from fireplace.utils import *
+from fireplace.controller import GameController
+from fireplace.ai.player import PlayerAI
+from fireplace.player import Player
+from fireplace.exceptions import GameOver
+
+logging.basicConfig(level=logging.INFO)
+logging.getLogger("fireplace").disabled = True
+logger = logging.getLogger("demo")
+
+
+"""
```

Comment:
All docstrings should be at the very top of the file so they can show up in `__doc__` (below the hashbang)